qcore/tests/test_helpers.py

```diff
@@ -190,19 +190,21 @@ def _stub_serializable_func():
 
 
 def test_object_from_string():
+    def check(st, expected):
```

Comment:
Can you test also that providing bytes in py3 throws a TypeError, and that providing some other random type throws a TypeError in both Python versions?